pkgs/applications/misc/synapse/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, intltool, pkgconfig, 
+  glib, libnotify, gtk3, libgee,
+  keybinder3, json_glib, zeitgeist, vala
```

Comment:
The comma goes on the next line:

```
{ stdenv, fetchurl, intltool, pkgconfig
, glib, ...
```

Also, why so short lines? We typically wrap lines at ~80 columns.